src/App/IntroPageAdmin/IntroPageAdmin.java

```diff
@@ -317,6 +318,31 @@ public void allEmployeeBtnAction(ActionEvent event) {
     }
 
     public void addEmployeeBtnAction(ActionEvent event) {
+        if (event.getSource() == addEmployeeBtn) {
+            if (getUserRole().matches("ADMIN_AUTH")) {
```

Comment:
Since the page is called `IntroPageAdmin` i.e. it will be visible to Admin only. We don't need to double check the admin status here.